### PR TITLE
feature(FR-393): add e2e test for maintenance page

### LINKED ISSUE #3041: E2E test for maintenance page
## Description

The `maintenance` page was recently refactored, and we need to ensure its functionality is stable. Set up and execute end-to-end (E2E) tests using `Playwright` to cover key user flows and edge cases.

## Plans

- I plan to write the test file by referring to the existing test files and the Playwright documentation.

JIRA Issue: FR-393

### PR DESCRIPTION
**Pull request by nowgnuesLee**
resolves #3041, [(FR-393)](https://lablup.atlassian.net/browse/FR-393)

**changes**
* Added test cases for `Rescan Images` and `Recalculate Usage` on the `maintenance` page.
* Added functions in `test-util-antd.ts` to retrieve notification text.
* Added `1-minute` timeout to rescan image test (1min)

**Checklist:**

- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: e2e/maintenance.test.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      await page.getByRole('button', { name: 'Rescan Images' }).click();
+
+      await expect(getNotificationMessageBox(page)).toContainText(
+        'Rescan Images',
+      );
+
+      await expect(getNotificationDescriptionBox(page)).toContainText(
+        'Rescanning...',
+      );
+      await expect(getNotificationDescriptionBox(page)).toContainText(
```
Comment (yomybaby) [inline e2e/maintenance.test.ts:40]: The rescan process can take longer than the default test timeout of 5 seconds, so this test can easily fail.

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by yomybaby — APPROVED** [review]
I added a commit to prevent unexpected notification closure.
